spec/src/main/asciidoc/license-alv2.asciidoc

```diff
@@ -29,7 +29,7 @@ Status: {revremark}
 
 Release: {revdate}
 
-Copyright (c) 2017-2019 Eclipse Microprofile Contributors
+Copyright (c) 2017-2020 MicroProfile Contributors
```

Comment:
Should we be changing Copyrights?  Maybe the original should be left as-is since through 2019, it was Eclipse MicroProfile.  And, then just add a new line for 2020-2020 MicroProfile?  Or, the follow the convention in the other files in this project, "..2020-2020 Contributors to the Eclipse Foundation".

Reply:
It needs to be MicroProfile Contributors

Reply:
This greatly matters because when we are ready to send the MicroProfile trademark, it is going to be MicroProfile across without the Eclipse. 

Therefore, MicroProfile nicely reverts to its original name under the WG umbrella, a huge perk. Questions, please bring them to tmw's after the Steering call is completed, there is the time for a community conversation to clear up any issues.